src/audioWorklet/index.js

```diff
@@ -6,7 +6,10 @@ const moduleSources = [
 ];
 const ac = p5sound.audiocontext;
 
-let initializedAudioWorklets = false;
+let ResolveOnWorkletLoaded;
```

Comment:
I don't think you need these changes unless you're not able to use the p5 `setup` function in your tests.